src/foolscap/banana.py

```diff
@@ -121,10 +121,10 @@ def populateVocabTable(self, vocabStrings):
         contents of this table.
         """
 
-        out_vocabDict = dict(zip(vocabStrings, range(len(vocabStrings))))
+        out_vocabDict = dict([(v,k) for k,v in enumerate(vocabStrings)])
         self.outgoingVocabTableWasReplaced(out_vocabDict)
```

Comment:
should use pep8 whitespacing `(v, k) for k, v` ...

Reply:
Done.